scilpy/tracking/local_tracking.py

```diff
@@ -299,30 +249,28 @@ def get_line_from_seed(tracker, mask, pos, pft_tracker, param):
         if ((len(line) > 1 and
              forward is not None and
              backward is not None and
-             len(line) >= param.min_nbr_pts and
-             len(line) <= param.max_nbr_pts)):
+             params.min_nbr_pts <= len(line) <= params.max_nbr_pts)):
             return line
-        elif (param.is_keep_single_pts and
-              param.min_nbr_pts == 1):
+        elif params.is_keep_single_pts and params.min_nbr_pts == 1:
             return [pos]
         return None
-    if ((param.is_keep_single_pts and
-         param.min_nbr_pts == 1)):
+    if ((params.is_keep_single_pts and
+         params.min_nbr_pts == 1)):
         return [pos]
     return None
 
 
-def _get_line(tracker, mask, pft_tracker, param, is_forward):
-    line = _get_line_binary(tracker, mask, param, is_forward)
+def _get_line(tracker, mask, params, is_forward):
+    line = _get_line_binary(tracker, mask, params, is_forward)
 
     while (line is not None and len(line) > 0 and
-           not tracker.isPositionInBound(line[-1])):
+           not tracker.is_position_in_bound(line[-1])):
         line.pop()
 
     return line
 
 
-def _get_line_binary(tracker, mask, param, is_forward):
+def _get_line_binary(tracker, mask, params, is_forward):
```

Comment:
Maybe we could come up with a new name for this one? It sounds like this returns a binary mask of some sort. It could maybe be merged with `_get_line`, actually.